robottelo/host_helpers/cli_factory.py

```diff
@@ -25,10 +25,12 @@
 from fauxfactory import gen_url
 
 from robottelo import constants
+from robottelo.cli.activationkey import ActivationKey
```

Comment:
No imports this should be accessed through satellite object.

`<sat_obj>.cli.ActivationKey`